src/main/cpp/RLBotInterface/RLBotMessages/PacketStructs/MatchDataPacket.hpp

```diff
@@ -3,23 +3,23 @@
 
 struct GoalInfo
 {
+	unsigned char			TeamNum;
 	PyStruct::Vector3		Location;
-	PyStruct::Rotator		Rotation;
 	PyStruct::Vector3		Direction;
-	PyStruct::Vector3		LocalMin;
-	PyStruct::Vector3		LocalMax;
-	PyStruct::Vector3		LocalExtent;
-	unsigned char			TeamNum;
+};
+
+struct BoostPad
+{
+	PyStruct::Vector3		Location;
+	bool					FullBoost;
```

Comment:
Spacing

Reply:
Will fix by using spaces instead of tabs for alignment. It lines up in visual studio because it displays tabs as 4 spaces instead of 8.